internal/db/migration/000005_rename_to_singular.up.sql

```diff
@@ -0,0 +1,7 @@
+BEGIN;
+
+ALTER TABLE "models" RENAME TO "model";
+ALTER TABLE "versions" RENAME TO "version";
+ALTER TABLE "t_models" RENAME TO "triton_model";
```

Comment:
Besides changing the table name, we also need to change the constraint name:
```
ALTER TABLE triton_model RENAME CONSTRAINT t_models_pkey TO triton_models_pkey
```
Also the foreign-key constraint name:
```
ALTER TABLE triton_model RENAME CONSTRAINT fk_tmodel_version TO fk_triton_model_version
```

Reply:
@pinglin yes I just keep origin name for simpler. But okie let change it for more consistency

Reply:
btw, beside constraint name we need rename index also. What do you think @pinglin ?